tests/vmi_multus_test.go

```diff
@@ -578,33 +584,33 @@ var _ = Describe("SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
+	createNetwork := func(networkName string, namespace string, networkAttachmentDefinition string) {
+		result := virtClient.RestClient().
+			Post().
+			RequestURI(fmt.Sprintf(postUrl, namespace, networkName)).
+			Body([]byte(fmt.Sprintf(networkAttachmentDefinition, networkName, namespace, sriovResourceName))).
+			Do()
+		Expect(result.Error()).NotTo(HaveOccurred())
```

Comment:
I think the error needs to be returned and not asserted here.
If you assert here, you will need to traceback the callers.

Returning the error and then asserting on it should be straight forward:
E.g.
```
expect(
    createNetwork("sriov", tests.NamespaceTestDefault, sriovConfCRD)
).NotTo(HaveOccured())
```

In general, I think it is much more appropriate to assert at the test level and not deep in the helpers.